UI/assets/js/adminDashboard.js

```diff
@@ -0,0 +1,75 @@
+
+/* Page elements */
+const username = document.getElementById('username');
+const requestStatusDropdown = document.getElementById('request-status-dropdown');
+const newRequestButton = document.getElementById('new-request-btn');
+const tableWrapper = document.getElementById('table-wrapper');
+const sortGroup = document.getElementById('sort-group');
+
+/* Set welcome message */
+const user = JSON.parse(window.localStorage.getItem('User'));
+console.log(user.firstName);
+username.innerHTML += `${user.firstName}.`;
```

Comment:
With the way I wrote it, the User object has to be thee because it's set after login. The user can't access the page otherwise. But I see your point, I'll add a check for that.